src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskLevelCollectionViewCell.xib

```diff
@@ -4,110 +4,106 @@
     <dependencies>
```

Comment:
This overrode all dark mode adjustments. (Resolve using "mine"?). I can re-apply the change, if you like.

Reply:
sure just do it. not overrode it on purpose.